mythril/analysis/modules/external_calls.py

```diff
@@ -85,7 +85,7 @@ def execute(statespace):
                     if "calldata" in target:
                         description += "an address provided as a function argument. "
                     else:
-                        description += "the address of the transaction sender. "
+                        description += "the address of the transaction senderself. "
```

Comment:
I think this is an accidental self.

Reply:
Fixed. What a *self-centered* mistake. \*badum-tss\* :drum: